src/style.css

```diff
@@ -0,0 +1,79 @@
+body {
+  padding: 0;
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+.search_form {
+  display: flex;
+  justify-content: left;
+  align-items: center;
+  background-color: #fff;
+  margin: 20px 0 0;
+}
+
+label {
+  width: 100%;
+}
+
+.search_input {
+  display: inline-block;
+  box-sizing: border-box;
+  font-family: Avenir-Book, sans-serif;
```

Comment:
move fonts to html

Reply:
fix it

Reply:
fix it 3-rd time

Reply:
![image](https://user-images.githubusercontent.com/99331793/188112590-6c3edf1b-d8cd-4fa0-993c-316296bb9d5a.png)